### PR TITLE
Flask Migration

### PR DESCRIPTION
**Pull request by Dj0ulo**
Contributor : @l9kd1

### CODE REVIEW

**Review by Drumor — COMMENTED** [review]
looks good

File: ictv/plugins/editor/capsules_page.py [hunk 1]
```diff
@@ -240,8 +242,8 @@ def render_page(self, channel, capsules=None):
 
         def get_data_edit_object(capsule):
             object = {'id': capsule.id, 'name': capsule.name}
-            object['from'] = c.validity_from.replace(microsecond=0).isoformat()
-            object['to'] = c.validity_to.replace(microsecond=0).isoformat()
```
Comment (anthonygego) [inline ictv/plugins/editor/capsules_page.py:244]: I guess this was a bug ?

**Review by anthonygego — APPROVED** [review]
LGTM

File: ictv/plugins/editor/capsules_page.py (hunk 1 above)
Comment (Dj0ulo) [reply ictv/plugins/editor/capsules_page.py:244]: yes indeed

**Pull request merged**